app/charts/shared/imputation.tsx

```diff
@@ -88,6 +88,59 @@ export const imputeTemporalLinearSeries = ({
   return dataSortedByX as Array<TemporalSeriesAfterImputationEntry>;
 };
 
+export const imputeTemporalLinearSeriesInverted = ({
+  dataSortedByY,
+}: {
+  dataSortedByY: Array<TemporalSeriesBeforeImputationEntry>;
+}): Array<TemporalSeriesAfterImputationEntry> => {
```

Comment:
Question: Should we also make it a bit more generic (`sortedData`) to re-use existing function? 🤔